api/routes.go

```diff
@@ -31,6 +31,7 @@ func InstallRoutes(proxyService *proxy.ProxyService, r *mux.Router) {
 	internalRouter.Handle("/metrics", promhttp.Handler())
 	internalRouter.HandleFunc("/status", status.GetStatus)
 	internalRouter.HandleFunc("/whoami", status.WhoAMI)
+	internalRouter.HandleFunc("/ui_metric", status.TrackUIMetric)
```

Comment:
Since this is going to be accessed by clients, putting it in its own versioned namespace under `/api` makes sense (like `/api/v1/metric/ui`)

Reply:
Should be POST-only to minimally prevent abuse.